deployability/modules/workflow_engine/examples/agent/aws/test-agent-basic-info.yaml

```diff
@@ -0,0 +1,205 @@
+version: 0.1
+description: This workflow is used to test agents deployment for DDT1 PoC
+variables:
+  agent-os:
+    - linux-redhat-7-amd64
+    - linux-redhat-7-arm64
+    - linux-redhat-8-amd64
+    - linux-redhat-8-arm64
+    - linux-redhat-9-amd64
+    - linux-redhat-9-arm64
+    - linux-centos-7-amd64
+    - linux-centos-7-arm64
+    - linux-centos-8-amd64
+    - linux-centos-8-arm64
+    - linux-debian-10-amd64
+    - linux-debian-10-arm64
+    - linux-debian-11-amd64
+    - linux-debian-11-arm64
+    - linux-debian-12-amd64
+    - linux-debian-12-arm64
+    - linux-ubuntu-22.04-amd64
+    - linux-ubuntu-22.04-arm64
+    - linux-ubuntu-18.04-amd64
+    - linux-ubuntu-18.04-arm64
+    - linux-ubuntu-20.04-amd64
+    - linux-ubuntu-20.04-arm64
+    - linux-oracle-9-amd64
+    - linux-amazon-2-amd64
+    - linux-amazon-2-arm64
+    - linux-amazon-2023-amd64
+    - linux-amazon-2023-arm64
+    - windows-desktop-10-amd64
+    - windows-server-2012r2-amd64
+    - windows-server-2016-amd64
+    - windows-server-2019-amd64
+    - windows-server-2022-amd64
+
+  macos-agent-os:
+    - macos-ventura-13-amd64
+    - macos-ventura-13-arm64
+    - macos-sonoma-14-amd64
+    - macos-sonoma-14-arm64
```

Comment:
Is it really needed to specify Ventura or Sonoma alongside the version number? There is no Ventura 14, Ventura implies the 13 version number

Reply:
These names are defined by the allocator
![image](https://github.com/wazuh/wazuh-qa/assets/89791732/91bebfc1-55c5-49dd-9edc-46e30b470abc)

Reply:
Mmmmm we can improve this in the future, but it will work for now